dist/index.js

```diff
@@ -36,28 +36,12 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
     });
 };
 Object.defineProperty(exports, "__esModule", ({ value: true }));
-const github = __importStar(__webpack_require__(438));
 const core = __importStar(__webpack_require__(186));
 const utils_1 = __webpack_require__(918);
 function run() {
     return __awaiter(this, void 0, void 0, function* () {
         try {
-            // SETUP
-            const github_token = core.getInput('github_token');
-            const octokit = github.getOctokit(github_token);
-            const { pull_request, repository } = github.context.payload;
-            const pull_number = (pull_request === null || pull_request === void 0 ? void 0 : pull_request.number) || 0;
-            const owner = (repository === null || repository === void 0 ? void 0 : repository.owner.login) || '';
-            const repo = (repository === null || repository === void 0 ? void 0 : repository.name) || '';
-            // ACTION
-            const reviewers = yield utils_1.listEligibleReviewers();
-            //octokit.github.io/rest.js/v18#pulls-request-reviewers
-            const result = yield octokit.pulls.requestReviewers({
-                pull_number,
-                reviewers,
-                owner,
-                repo
-            });
+            const result = yield utils_1.requestReviewer();
```

Comment:
`yield` always scares me heh

Reply:
hehehe, that's what happens with all async/awaits behind the scenes, the transpiler transforms it in generator functions.